gtsfm/data_association/data_assoc.py

```diff
@@ -196,6 +204,68 @@ def run(
 
         return connected_data, data_assoc_metrics
 
+    def run_triangulation(
+        self,
+        cameras: Dict[int, gtsfm_types.CAMERA_TYPE],
+        tracks_2d: List[SfmTrack2d],
+    ) -> Tuple[List[Delayed], List[Delayed], List[Delayed]]:
+        """Performs triangulation of 2D tracks in parallel.
+
+        Refs:
+        - https://docs.dask.org/en/stable/delayed-best-practices.html#compute-on-lots-of-computation-at-once
+        - https://docs.dask.org/en/stable/delayed-best-practices.html#avoid-too-many-tasks
+
+        Args:
+            cameras: list of cameras wrapped up as Delayed.
```

Comment:
nit: can we capitalize "List of cameras", since Google style dictates this

Reply:
Done

Reply:
Marking as resolved, thanks!